### DISCUSSION TITLE
Back/forward buttons in mouse layer

### DISCUSSION THREAD

**Discussion by jurf** [manna-harbour/miryoku#212] [Q&A] [answered]
Hey,

I really enjoy Miryoku so far, but I dearly miss back/forward buttons for navigation (`KC_WWW_BACK` and `KC_WWW_FORWARD`). I noticed there are two blank spots in the mouse layer so I added them there. Many mice have these buttons so it seems a fitting place too. What do you think?

**Comment by manna-harbour** [comment, marked as answer]
In general I'd just use Alt-left/right from Nav (or the vimium keybinds from Base).

For a physical mouse it might be useful as you have one hand on the mouse and you don't have 2 hands on the keyboard, but I don't think it'll be very useful on a mouse layer, unless you use it to click links and use layer lock. But to click links I'd prefer to use vimium. 

It might be useful on Button with an integrated pointing device, if you have layer activation on movement enabled, and you want to browse with one hand.

The unused keys on Mouse / Button will be used for drag scroll and sniping in future (when those features make it to QMK master), but you may as well make use of them for now!

**Reply by jurf** [reply]
Thanks for the reply! Hm, makes sense if there is a better use for them. Although I still think they're useful, as they are quite universal and don't require chording.

**Reply by manna-harbour** [reply]
If you'd like to try it you can substitute a custom layer with https://github.com/manna-harbour/miryoku/discussions/85.
